corehq/form_processor/models/cases.py

```diff
@@ -834,6 +835,16 @@ def save(self, *args, with_tracked_models=False, **kw):
         except DatabaseError as e:
             raise CaseSaveError(e)
 
+    def delete(self, leave_tombstone=False):
```

Comment:
Would be great to have a test that deletes a case using a tombstone.

Reply:
Would it make sense to have the default be `leave_tombstone=True`? It seems like we would want that most of the time, excluding tests, which we can handle with a different workaround.